config/locales/en.yml

```diff
@@ -115,10 +115,10 @@ en:
     application_status: Status
 
   table_headings: &TABLE_HEADINGS
-    applicant_name: Applicant
-    reference: Ref. no.
-    days_passed: Business days since applications were received
-    time_passed: Days passed
+    applicant_name: Applicants name
+    reference: Reference number
+    days_passed: Business days since application was received
```

Comment:
Revert this back to the plural - it's being used in the dropdown mini report table